js/index.js

```diff
@@ -0,0 +1,204 @@
+        const template = `
+        <div class="card__item" data-id="${product.id}"">
+            <div class="card__image">
+                <img src='${product.images[0]}' />
+            </div>
+            <div class="card__info">
+                <div class="card__comment">
+                    <span>&#9733;&#9733;&#9733;&#9733;&#9734;</span>
+                    <span><a href="#">${product.id} reviews</a></span>
+                </div>
+                <div class="card__title">
+                    <p>
+                        ${product.title}
+                    </p>
+                </div>
+                <div class="card__price">
+                    <span class="card__price__current-price"> ${product.price} ₴</span
+                    >
+                </div>
+             </div>
+        </div>`;
+
+        return template;
+    };
+
+    const templateProductInformation = (product) => {
+        const templateProductInfo = `
+            <div class="info__item" data-id = ${product.id}>
+                <div class="info__item__category">
+                    <span>${product.category.name}</span>
+                </div>
+                <div class="info__item__title">
+                    <h3>${product.title}</h3>
+                </div>
+                <div class="info__item__image">
+    
+                    <img
+                        src="${product.images ? product.images[0] : './img/no-image.png'}"
+                        alt=""
+                    />
+                </div>
+    
+                <div class="info__item__commment">
+                    <div>
+                        <span>&#9733;&#9733;&#9733;&#9733;&#9734;</span>
+                        <span>${product.id} ratings</span>
+                    </div>
+    
+                    <div class="info__item__id">Code: 2220${product.id}</div>
+                </div>
+    
+                <div class="info__item__price">
+                    <span class="info__item__price-current">${product.price} ₴</span>
+                </div>
+                <div class="info__item__characteristics">
+                    <button>About</button>
+                    <button>Characteristics</button>
+                    <button>Reviews</button>
+                </div>
+                <div class="info__item__about">
+                    <p>
+                        ${product.description}
+                    </p>
+                </div>
+                <div class="info__item__buttons">
+                    <button class="button__add">Add to basket</button>
+                </div>
+            </div>
+        `;
+
+        return templateProductInfo;
+    };
+
+    const getProductData = async (offset = 0, limit) => {
+        let data = null;
+
+        try {
+            let responce = await fetch(`https://api.escuelajs.co/api/v1/products?offset=${offset}&limit=${limit}`);
+            data = await responce.json();
+
+            if (responce.status != 200) {
```

Comment:
NEVER USE NON-STRICT equality / non-equality 
```suggestion
            if (responce.status !== 200) {
```